src/rulesets/oas/functions/__tests__/runtimeExpression.test.ts

```diff
@@ -0,0 +1,178 @@
+import { DiagnosticSeverity } from '@stoplight/types';
+import { functions } from '../../../../functions';
+import { runtimeExpression } from '../runtimeExpression';
+import { DocumentInventory } from '../../../../documentInventory';
+import { Document } from '../../../../document';
+import * as Parsers from '../../../../parsers';
+import { isOpenApiv3, RuleType, Spectral } from '../../../..';
+import { rules } from '../../index.json';
+
+function runRuntimeExpression(targetVal: any) {
+  const doc = new Document(JSON.stringify(targetVal), Parsers.Json);
+
+  return runtimeExpression.call(
+    { functions },
+    targetVal,
+    null,
+    { given: ['paths', '/path', 'get', 'responses', '200', 'links', 'link', 'parameters', 'param'] },
+    { given: null, original: null, documentInventory: new DocumentInventory(doc, {} as any), rule: {} as any }, // TODO
```

Comment:
You probably don't need to bother about this `todo` comment.
As a part of the V6 release, a custom function will receive some lifting here and there.